DigitalLearningSolutions.Web.Tests/ViewModels/MyAccount/MyAccountViewModelTests.cs

```diff
@@ -27,6 +28,11 @@ public void MyAccountViewModel_AdminUser_and_DelegateUser_populates_expected_val
                 returnedModel.ProfilePicture.Should().BeEquivalentTo(adminUser.ProfileImage);
                 returnedModel.DelegateNumber.Should().BeEquivalentTo(delegateUser.CandidateNumber);
                 returnedModel.User.Should().BeEquivalentTo(adminUser.EmailAddress);
+                returnedModel.JobGroup.Should().BeEquivalentTo(delegateUser.JobGroupName);
```

Comment:
I'm wondering if it's worth separating the customPrompts population into a separate test to make it easier to debug if anyone breaks it in the future?